src/Models/Message.php

```diff
@@ -80,6 +84,69 @@ public function scopeOfConversation($query, Conversation $conversation)
         );
     }
 
+    /**
+     * Mark the notification as read.
+     *
+     * @param  \OwowAgency\Gossip\Models\Contracts\HasConversationContract  $model
+     * @return void
+     */
+    public function markAsRead(HasConversationContract $model)
+    {
+        $this->users()->syncWithoutDetaching($model);
+    }
+
+    /**
+     * Mark the notification as unread.
+     *
+     * @param  \OwowAgency\Gossip\Models\Contracts\HasConversationContract  $model
+     * @return void
+     */
+    public function markAsUnread(HasConversationContract $model)
+    {
+        $this->users()->detach($model);
+    }
+
+    /**
+     * Determine if a notification has been read.
+     *
+     * @param  \OwowAgency\Gossip\Models\Contracts\HasConversationContract  $model
+     * @return bool
+     */
+    public function read(HasConversationContract $model = null): bool
```

Comment:
Would `hasRead` be a better name?

Reply:
Not sure. Now it has the same naming conventions [as Laravel does](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Notifications/DatabaseNotification.php#L87).